index.bs

```diff
@@ -2581,6 +2581,38 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn> that is greater than zero. This
+means that even when the {{AudioNode}} is fed silence, non-silent audio can be
+output.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] when and only when it
+	is playing for at least part of the current rendering quantum.
+- A {{DelayNode}} in a cycle is actively processing only when the absolute value
+	of any output sample for the current [=render quantum=] is greater than or equal
+	to \( 2^{-126} \).
+- A {{ScriptProcessorNode}} is [=actively processing=] when its input or output is
+	connected.
+- An {{AudioWorkletNode}} is [=actively processing=] when its
+	{{AudioWorkletProcessor}}'s {{[[callable process]]}} is <code>true</code> and
+	either its [=active source=] flag is <code>true</code> or any {{AudioNode}}
+	connected to one of its inputs is [=actively processing=].
+- All other {{AudioNode}}s start [=actively processing=] when any
+	{{AudioNode}} connected to one of its inputs is [=actively processing=], and
```

Comment:
Need to say something about MediaElement/MediaStream sources which aren't AudioScheduledSourceNodes and don't have inputs.

Reply:
Yes, thanks.  How about these?

"An AudioScheduledSourceNode or MediaElementAudioSourceNode is actively
processing if and only if it or its `mediaElement` is playing for at least part of the current rendering quantum."

"A MediaStreamAudioSourceNode or MediaStreamTrackAudioSourceNode is actively
processing when the MediaStreamTrack is `live`, `unmuted`, and `enabled`."

Reply:
Fixed, doing something like @karlt's suggestions.